gateway/src/test/java/io/zeebe/gateway/ResponseMapperTest.java

```diff
@@ -58,4 +60,16 @@ public void shouldTestDeployWorkflowMapping() {
               assertTrue(factory.contains(workflow));
             });
   }
+
+  @Test
+  public void shouldTestCreateWorkflowInstanceMapping() {
```

Comment:
Personally I don't see value in these tests either as they are more boiler plate then what they add as value compared to the tests in the client test suite. Also you now had to change the implementation of the 3 entities just to make it easier to create the boiler plate for these tests. Not really liking it.